src/Messages/Interactions/TSInfoMessage.h

```diff
@@ -16,8 +17,7 @@ typedef NS_ENUM(NSInteger, TSInfoMessageType) {
     TSInfoMessageTypeDisappearingMessagesUpdate
 };
 
-+ (instancetype)userNotRegisteredMessageInThread:(TSThread *)thread
-                                     transaction:(YapDatabaseReadWriteTransaction *)transaction;
++ (instancetype)userNotRegisteredMessageInThread:(TSThread *)thread;
```

Comment:
This is some minor unrelated cleanup.  I noticed that we were needlessly passing around a transaction to this method.